src/features/game/components/Withdrawn.tsx

```diff
@@ -11,21 +12,28 @@ export const Withdrawn: React.FC = () => {
 
   const addTokenToMetamask = async () => {
```

Comment:
This is too generic name, also we are not accepting any token object with it's info... 😄 
`addSflToMetamaskList`  would be more accurate. Also easy to find reference in IDE's... :)

Reply:
Modifying `addSflToMetamaskList` per request!